.github/workflows/deploy.yml

```diff
@@ -16,4 +16,4 @@ jobs:
       contents: read
     uses: ./.github/workflows/reusable-deploy.yml
     with:
-      stage: ${{ matrix.stage }}​
+      stage: ${{ matrix.stage }}
```

Comment:
@iamazeem what is the change here. I can't see it in the diff...

Reply:
Not sure. There was some invisible character at the end of this line.

Reply:
Oh I see, there's some kind of hidden character?  How bizarre.